src/pages/dataTableUtilities/getColumns.js

```diff
@@ -583,7 +590,7 @@ const COLUMNS = () => ({
     type: COLUMN_TYPES.STRING,
     key: COLUMN_KEYS.NAME,
     title: tableStrings.question,
-    textAlign: 'left',
+    alignText: 'left',
```

Comment:
the rest of these are offroads.. but will help others in the future when they puzzle over funny looking columns